src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -564,18 +590,18 @@
          [:f>
           (fn []
             (let [reply (<sub [:chats/reply-message])
+                  suggestions (<sub [:chat/mention-suggestions])
                   {window-height :height} (rn/use-window-dimensions)
                   {:keys [keyboard-shown keyboard-height]} (rn/use-keyboard)
-
                   max-y (- window-height (if (> keyboard-height 0) keyboard-height 360) (:top insets)) ; 360 - default height
                   max-height (- max-y 56 (:bottom insets))  ; 56 - top-bar height
-                  added-value (if reply 38 0) ; increased height of input box needed when reply
-                  min-y (+ min-y added-value)
+                  added-value (if (and (not (seq suggestions)) reply) 38 0) ; increased height of input box needed when reply
```

Comment:
are we sure we want to hide reply in case of suggestions ?

Reply:
@flexsurfer in case of suggestions we don't need the `added-value` to `y` because the bottom buttons view will not be rendered, so that will make up for the difference.